src/defaults.mligo

```diff
@@ -10,6 +10,7 @@ let default_config (data : initial_config_data) : config = {
   decision_lambda = (fun (proposal, extras : proposal * contract_extra) -> (([] : (operation list)), extras));
 
   quorum_threshold = data.quorum_threshold;
+  fixed_proposal_fee_in_token = 0n;
```

Comment:
@sras this is a bug reported in #206, this should be:
```suggestion
  fixed_proposal_fee_in_token = data.fixed_proposal_fee_in_token;
```
except that `initial_config_data` doesn't have that field.

If @rinn7e hasn't started working on that issue yet can you self-assign it and fix it here by adding a commit just for it?
I think it will avoid a merge conflict and be quicker to proceed this way.

Reply:
Have updated